backend-api/src/functions/testUtils/mockJwtBuilder.ts

```diff
@@ -111,6 +114,15 @@ export class MockJWTBuilder {
   };
 }
 
+const MOCK_PRIVATE_KEY = {
```

Comment:
Please may we add a comment next to this saying:
- How this was generated
- That this is only used in unit tests and not used in any AWS environment

It will get flagged on a github secret scan and the comments really help people that review them :)

Reply:
I've renamed this to MOCK_SIGNING_KEY and left a small comment to say it's only used in unit tests